File: camps/ptz_winter14/day7/d/gen.pl
#!/usr/bin/perl

my $n = 35000;
my $m = 35000;
print "$n $m 0\n";
for(1..$n) {
    print "0 0 $_ 0\n";
}
for(1..$m) {
    print "0 0 1\n";
}

File: camps/ptz_summer14/day5/f/gen.py
#!/usr/bin/python
from random import randint, shuffle
from sys import argv

n = int(argv[1])

print n,n
a = list(range(n))
shuffle(a)
for i in range(n-1):
#     print a[i]+1, a[i+1]+1
#     print i+2, randint(1, i+1)
    print a[i+1]+1, a[randint(0, i)]+1
for i in range(n-1):
    print i+2, (i+2)/2
for i in range(1000):
    print randint(1, n), randint(1, n)
